.github/workflows/push-deploy-website.yml

```
@@ -0,0 +1,57 @@
+#
+# NOTE!
+#
+# Please read the README.md file in this directory that defines what should 
+# be placed in this file
+#
+##############################################################################
+##############################################################################
+
+name: PUSH Workflow - Website Deployment
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)